modules/pubmaticBidAdapter.js

```diff
@@ -544,9 +550,9 @@ function _createVideoRequest(bid) {
       videoObj.w = parseInt(bid.mediaTypes.video.playerSize[0], 10);
       videoObj.h = parseInt(bid.mediaTypes.video.playerSize[1], 10);
     }
-    if (bid.params.video.hasOwnProperty('skippable')) {
+    if (videoData.hasOwnProperty('skippable')) {
       videoObj.ext = {
-        'video_skippable': bid.params.video.skippable ? 1 : 0
+        'video_skippable': videoData.skippable ? 1 : 0
```

Comment:
I think ORTB calls this 'skip'

Reply:
Yes. Agreed. Removing this and adding support for skip as boolean property.